tests/qos/files/qos_params.gb.yaml

```diff
@@ -414,7 +414,7 @@ qos_params:
                 wm_q_wm_all_ports:
                     ecn: 1
                     pkt_count: 3000
-                    pkts_num_margin: 1024
+                    pkts_num_margin: 3072
```

Comment:
Can you elaborate why we need this much higher margin? seems the watermark margin will be 3072*384(cell_size) = around 1.2M bytes?

Reply:
The Qwatermark goes up in a very specific steps(thanks to Zhixin):
```
// Lossy 100G VOQ SMS buffer size quantization thresholds
    la_voq_cgm_quantization_thresholds m_voq_sms_thresh_100g = {.thresholds = {(50 * 384),
                                                                               (96 * 384),
                                                                               (192 * 384),
                                                                               (512 * 384),
                                                                               (1 * 1024 * 384),
                                                                               (4 * 1024 * 384),
                                                                               (6 * 1024 * 384),
                                                                               (7 * 1024 * 384),
                                                                               (8 * 1024 * 384),
                                                                               (9 * 1024 * 384),
                                                                               (10 * 1024 * 384),
                                                                               (12 * 1024 * 384),
                                                                               (14 * 1024 * 384),
                                                                               (15 * 1024 * 384),
                                                                               (16000 * 384)}}; 
```
So basically it goes up in 50, 96, 192, 512 buffers and so on. For 3000 packets, the WM will be 4*1024*384. not the one currently calculated in the script, which is based on 3*1024*384. In order to accomodate this we increase the margin for this test.

Reply:
okay to merge.